### DISCUSSION TITLE
Gallery not showing despite using template?

### DISCUSSION THREAD

**Discussion by mcgregorian1** [greenelab/lab-website-template#207] [Q&A] [answered]
I think this must be simple, but I'm unable to find out what's going on. I've added a Funding and affiliations section on the [Team](https://deadlabcary.github.io/lab-website/team/) page similar to the GreeneLab website itself, but not only will the gallery not populate correctly, but the images themselves are not showing. Here is the [github site](https://github.com/deadLabCary/lab-website). I thought it might be because of how the gallery is defined, especially because with other websites in your examples, the gallery is defined using different language (for example, [this site's version](https://raw.githubusercontent.com/QuantMarineEcoLab/quantmarineecolab.github.io/main/team/index.md)).

What do you suggest? Thank you!

**Comment by vincerubinetti** [comment]
I'm out of office today and on my phone right now.

But as for the grid not working, check this docs page:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/grid

You need to put an empty line between each image if using markdown in the grid.

As for the images not showing, your markdown syntax of an image wrapped in a link seems correct, but maybe check that it is valid markdown.

**Reply by vincerubinetti** [reply]
Also, the Greene Lab site is still using a slightly older version where we had a specific image gallery component, which I replaced with the more generic and flexible grid component.

Look at the examples in that grid docs page or how the template does it [here](https://github.com/greenelab/lab-website-template/blob/main/team/index.md).

The Greene Lab site shouldn't be used as a direct reference over the template and docs.

**Reply by mcgregorian1** [reply]
Thanks! The line between each image worked, and thank you for linking the grid documentation. The images still do not show, and I've verified that the use of the markdown syntax is fine. That being said, the `{% include figure.html ... %}` syntax you have in the docs does indeed show the image.

Do you know how to add a link to that syntax? From what I can see I'd need to add something explicitly to `_includes/figure.html`; is that correct?

**Reply by vincerubinetti** [reply, marked as answer]
The figure component already accepts a link parameter:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/components/figure

As for the linking issue, I'm imagining it must be a problem with you not including your base url in the link. See this part of the docs:

https://greene-lab.gitbook.io/lab-website-template-docs/basics/write-basic-content#advanced

This page/section may need to be made a little clearer.

Basically, try removing the first slash in your image urls to make them relative.

**Reply by mcgregorian1** [reply]
Ah, perfect, thanks for linking the figure docs. 

I tried updating the base url but it still didn't work, for whatever reason. I think you're right, though, that this is some issue with my path. However, since the html-based syntax you have does work, I'll just go with that. Thank you!
